xivo/tenant_helpers.py

```diff
@@ -145,10 +147,13 @@ def tenant_uuid(self):
     def user_uuid(self):
         return self._token_dict['metadata'].get('uuid')
 
-    def visible_tenants(self, tenant_uuid=None):
-        if not tenant_uuid:
+    def visible_tenants(self, tenant_uuid=UNSET):
+        if tenant_uuid is UNSET:
```

Comment:
I'm not familiar with this kind of pattern `UNSET = object()`. The following example would also work:
```
if not tenant_uuid:
  tenant_uuid = self.tenant_uuid
if not tenant_uuid:
  return []
```
I you prefer the UNSET, I'm OK, no need to argument
Just close the comment

Reply:
This pattern allow to differenciate `visible_tenant(None)` from `visible_tenant()`.
I often use it, but it looks useless here.